src/main/resources/public/jsx/DatasetForm.jsx

```diff
@@ -117,7 +118,14 @@ class DatasetForm extends React.Component {
                                         </div>
                                 )}
                             </FormGroup>
-
+                                
+                            <FormGroup>
+                                <Label htmlFor="private" value={ t('dataset.label.private') } />                 
```

Comment:
There is no `id` bound to this `htmlFor`

Reply:
`htmlFor` removed